example/js/directives/angular-pdf.js

```diff
@@ -165,7 +166,7 @@
           }
 
           if (url && url.length) {
-            pdfLoaderTask = PDFJS.getDocument(params, null, null, scope.onProgress);
+            pdfLoaderTask = PDFJS.getDocument(params);
```

Comment:
Hi Thanks for the patch! 

Do you know how else we can refer to `scope.onProgress` if we remove it? See in the [example code](https://github.com/sayanee/angularjs-pdf/blob/be1ab8cf11ddd1c6f2bfb7f7861fb2d758b6f1a5/example/js/controllers/docCtrl.js#L21-L23).